src/notifications/use-push-notifications.tsx

```diff
@@ -47,7 +47,8 @@ export const NotificationContextProvider: React.FC = ({children}) => {
   const [fcmToken, setFcmToken] = useState<string>();
   const {mutation: registerMutation} = useRegister();
   const mutateRegister = registerMutation.mutate;
-  const {query: configQuery, mutation: configMutation} = useConfig();
+  const {query: configQuery, mutation: configMutation} =
```

Comment:
Perhaps use getConfigQuery or getNotificationConfig as well updateNotificationConfig here?

Reply:
I think leaving it as-is would be preferable, to keep it aligned with tanstack-query standards. It would be weird if we call `updateNotificationConfig.mutate`. It might be better with `getConfigQuery.data` but it doesn't add that much value anyway. So keeping it as-is for now, unless we want to start renaming mutations and queries. :)

Reply:
OK, but I was more thinking:

```ts
const {query: getNotificationConfig, mutation: updateNotificationConfig} = useConfig();
```

not: `updateNotificationConfig.mutate` but I see that it is a standard

Reply:
Thanks Jorge, Sorry I got a typo there on my comment hahaha.